prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics-release.yaml

```diff
@@ -75,6 +79,8 @@ periodics:
       args:
       - make
       - test-e2e-kind-multi-repo
+      - 'E2E_OCI_PROVIDER=local'
```

Comment:
Do we need to set it explicitly? `local` is the default value.

Reply:
We need to override the value set in the parent template, which is set to `gar` so that it applies to all of the other periodic jobs (which run on GKE)

Reply:
ACK

Reply:
Alternatively we could update the `test-e2e-kind-multi-repo` target in the repo to always set the providers to local